pyaedt/aedt_logger.py

```diff
@@ -579,7 +579,8 @@ def _log_on_dekstop(self, message_type, message_text, level=None, proj_name=None
         if not level:
             level = "Design"
 
-        assert level in message_levels, "Message level must be `Design', 'Project', or 'Global'."
+        if level not in message_levels:
+            raise ValueError("Message level must be 'Design', 'Project' or 'Global'.")
```

Comment:
```suggestion
            raise ValueError("Message level must be 'Design', 'Project', or 'Global'.")
```
We always use an Oxford comma, which say to place a comma before an "or" or an "and" in a list of three or more items. What was originally there was correct.